dictionaries/software-terms/src/software-terms.txt

```diff
@@ -254,6 +257,7 @@ Cholesky
 class
 clint
 Clojure
+cloneable
```

Comment:
@ccoVeille,

Adding it here causes other dictionaries that use software-terms as a list of known words to split compound that were previously unknown.

This is related to the spelling corrections lists as well.